xapian-applications/omega/templates/query

```diff
@@ -153,3 +153,4 @@ $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </body>
 </html>
 $log{query.log}
+$log{search.log,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}$.$csv{$query,1}$.$csv{$hitlist{$id$if{$ne{$hit,$sub{$last,1}},$.}},1}$.$topdoc}
```

Comment:
I think adding an `$if{}` (based on `$opt{clicklog}`) here should solve the problem of unconditional logging.

However, I don't get how we might end up with non-matching query ids in `search.log` and `clicks.log` because we don't log query ids in `clicks.log`.